includes/organization_config.php

```diff
@@ -11,52 +11,52 @@ function organization_config_mnu() {
 		$org_options['organization_state'] = isset($_POST['org_state']) && !empty($_POST['org_state']) ? stripslashes_deep($_POST['org_state']) : '';
 		$org_options['organization_zip'] = isset($_POST['org_zip']) && !empty($_POST['org_zip']) ? stripslashes_deep($_POST['org_zip']) : '';
 		$org_options['organization_country'] = isset($_POST['org_country']) && !empty($_POST['org_country']) ? stripslashes_deep($_POST['org_country']) : '';
-		$org_options['organization_country'] = isset($_POST['org_country']) && !empty($_POST['org_country']) ? $_POST['org_country'] : '';
-		$org_options['contact_email'] = isset($_POST['email']) && !empty($_POST['email']) ? $_POST['email'] : '';
-		$org_options['expire_on_registration_end'] = isset($_POST['expire_on_registration_end']) && !empty($_POST['expire_on_registration_end']) ? $_POST['expire_on_registration_end'] : '';
-		$org_options['event_page_id'] = isset($_POST['event_page_id']) && !empty($_POST['event_page_id']) ? $_POST['event_page_id'] : '';
-		$org_options['return_url'] = isset($_POST['return_url']) && !empty($_POST['return_url']) ? $_POST['return_url'] : '';
-		$org_options['cancel_return'] = isset($_POST['cancel_return']) && !empty($_POST['cancel_return']) ? $_POST['cancel_return'] : '';
-		$org_options['notify_url'] = isset($_POST['notify_url']) && !empty($_POST['notify_url']) ? $_POST['notify_url'] : '';
-		$org_options['events_in_dasboard'] = isset($_POST['events_in_dasboard']) && !empty($_POST['events_in_dasboard']) ? $_POST['events_in_dasboard'] : '';
-		$org_options['default_mail'] = isset($_POST['default_mail']) && !empty($_POST['default_mail']) ? $_POST['default_mail'] : '';
-		$org_options['payment_subject'] = isset($_POST['payment_subject']) && !empty($_POST['payment_subject']) ? $_POST['payment_subject'] : '';
+		$org_options['organization_country'] = isset($_POST['org_country']) && !empty($_POST['org_country']) ? sanitize_text_field($_POST['org_country']) : '';
+		$org_options['contact_email'] = isset($_POST['email']) && !empty($_POST['email']) ? sanitize_text_field($_POST['email']) : '';
+		$org_options['expire_on_registration_end'] = isset($_POST['expire_on_registration_end']) && !empty($_POST['expire_on_registration_end']) ? sanitize_text_field($_POST['expire_on_registration_end']) : '';
+		$org_options['event_page_id'] = isset($_POST['event_page_id']) && !empty($_POST['event_page_id']) ? sanitize_text_field($_POST['event_page_id']) : '';
+		$org_options['return_url'] = isset($_POST['return_url']) && !empty($_POST['return_url']) ? sanitize_text_field($_POST['return_url']) : '';
```

Comment:
```suggestion
		$org_options['return_url'] = isset($_POST['return_url']) && !empty($_POST['return_url']) ? esc_url($_POST['return_url']) : '';
``` would probably be more appropriate for all the URL fields. See https://developer.wordpress.org/reference/functions/esc_url/ if you want (it's a bit strange that it's not one of the `sanitize_*` functions)

Reply:
Thanks. I found a couple more places this was needed.